aspnetcore/fundamentals/configuration/index.md

```diff
@@ -217,7 +217,28 @@ setx Logging__1__Level=Information
 
 ### Environment variables set in launchSettings.json
 
-Environment variables set in *launchSettings.json* override those set in the system environment.
+Environment variables set in *launchSettings.json* override those set in the system environment. For example, the ASP.NET Core web templates generate a *launchSettings.json* file that sets the [Kestrel endpoint configuration](#kestrel) to:
+
+```json
+"applicationUrl": "https://localhost:5001;http://localhost:5000"
+```
+
+The preceding setting overrides values set in the environment.
+
+### Escape environment variables on Linux
+
+On Linux, the value of URL environment variables must be escaped so `systemd` can parse it. Use the linux tool `systemd-escape` which yields `http:--localhost:5001`
+ 
+ ```cmd
+ groot@terminus:~$ systemd-escape http://localhost:5001
+ http:--localhost:5001
+ ```
+
```

Comment:
@SidShetye please review. I took this from https://github.com/dotnet/aspnetcore/issues/15011#issuecomment-566801447